vc_zoom/indico_vc_zoom/__init__.py

```diff
@@ -5,7 +5,27 @@
 # them and/or modify them under the terms of the MIT License;
 # see the LICENSE file for more details.
 
+from indico.core import signals
+from indico.core.db import db
+from indico.modules.logs import EventLogRealm, LogKind
 from indico.util.i18n import make_bound_gettext
+from indico_vc_zoom.task import refresh_meetings
 
 
 _ = make_bound_gettext('vc_zoom')
+
+
+@signals.event.updated.connect
```

Comment:
Is there any difference between the decorator directly or connecting through the IndicoPlugin class?

Reply:
If you use the decorator, it will run at import time; if you call it in the plugin's init method it will only run when the plugin is actually loaded

Reply:
Yes, I moved into this file to cope with that - is there anything against having it at import time?

Reply:
I would put stuff in the plugin like we do in the other plugins as well; usually `__init__.py` of plugins doesn't contain anything but the i18n binding and (if needed) binding the `import_tasks` signal (i don' remember if that was needed or if i just didn't think about putting it in the plugin when adding it for the first time to ap lugin)